client/src/sagas/types.js

```diff
@@ -1,4 +1,21 @@
-export const LOGIN = 'LOGIN';
-export const LOGIN_ASYNC = 'LOGIN_ASYNC';
-export const REGISTER = 'REGISTER';
-export const REGISTER_ASYNC = 'REGISTER_ASYNC';
+// "reducer" actions should never be used outside of the reducers and the sagas pushing to the reducers
```

Comment:
Is there something I could read to learn more about reducers/sagas?

Reply:
[This](https://react-redux.js.org/introduction/basic-tutorial) should teach you about vanilla redux, and [this](https://redux-saga.js.org/) should tell you a bit more about sagas.